paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -173,6 +173,7 @@ class PaparazziPlugin : Plugin<Project> {
               nativePlatformFileCollection.singleFile.absolutePath
             test.systemProperties["paparazzi.test.record"] = isRecordRun.get()
             test.systemProperties["paparazzi.test.verify"] = isVerifyRun.get()
+            test.systemProperties["kotlinx.coroutines.main.delay"] = true
```

Comment:
spoke offline, slight concern about how to catch if this property key changes other than the backing test (which will fail, but might not obvious to why?)

Reply:
Looks like it is a documented property in coroutines 1.6.0:
https://github.com/Kotlin/kotlinx.coroutines/blob/master/CHANGES.md#version-160
Strangely, according to that wording it should be defaulting to true.